doc/configuration.rst

```diff
@@ -713,6 +713,8 @@ If you customize return codes, prefer to set them in range 128 to 255 to avoid
 conflicts. Return codes in range 0 to 127 are reserved for possible future use
 by Luigi contributors.
 
+If you run *luigi.build()/luigi.run()* programmatically, you can enable a detailed response of type :class:`~luigi.execution_summary.LuigiRunResult` by sending ``detailed_summary=True`` to these functions. This detailed response has an attribute(``status``) which is a status code that contains information about the progress. The list of possible status codes can be found in :class:`~luigi.execution_summary.LuigiStatusCode`.
```

Comment:
I would remove this whole part:

 > This detailed response has an attribute(``status``) which is a status code that contains information about the progress. The list of possible status codes can be found in :class:`~luigi.execution_summary.LuigiStatusCode

Because you already link to `LuigiRunResult` which you've already written excellent class documentation for  which explains all attributes.

------

Hey wait, I just noticed something important. Why do this documentation live in `configuration.rst`? This has nothing to do with configuration. I think whatever you have in `running_luigi.rst` is enough. Less is more, let's just keep the quality of the "less" high. :)

Reply:
> I think whatever you have in running_luigi.rst is enough

😄 Remember you asked me to link the existing `retcodes` docs to this new development. This is why I added this in `configuration.rst`.

- Getting rid of `configuration.rst` changes for now. Let me know if you want them back 🤖